### ISSUE TITLE
Feedback attribute: What does it do?

### ISSUE THREAD

**Issue by Vadorequest** [google/chatbase-node#26]
I do not understand the usage of the "feedback" attribute

`.setAsFeedback() // sets the message as feedback from the user`

Could you share a meaningful use case? I couldn't find anything relevant in the official chatbase documentation either.

**Comment by florent1933** [comment]
Same for me.

Did u find something @Vadorequest?

Best regards,

**Comment by Vadorequest** [comment]
Nope. I tried it out and don't see any change on the dashboard. I won't use this to store customer's replies.

**Comment by florent1933** [comment]
Thanks @Vadorequest
